services/backend/src/config/index.js

```diff
@@ -47,6 +47,11 @@ const { DB_URL_KONE, DB_URL_USER, SECRET_TOKEN, SIS_DB_URL, SIS_UPDATER_URL, CRY
 
 const rootOrgId = process.env.ROOT_ORG_ID || 'hy-university-root-id'
 
+const languageCenterViewEnabled =
+  process.env.LANGUAGE_CENTER_VIEW_ENABLED == null ? true : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'
```

Comment:
Would it make also like this?
`const languageCenterViewEnabled = !process.env.LANGUAGE_CENTER_VIEW_ENABLED
  ? true
  : process.env.LANGUAGE_CENTER_VIEW_ENABLED === 'true'`

Reply:
This solution would give the wrong result, as process.env.LANGUAGE_nönönö is a string, so if process.env.LANGUAGE_nönönö is set, this (!process.env.LANGUAGE_nönönö) will be evaluated as false, no matter what string that is passed to it.

The evaluation with process.env.LANGUAGE_nönönö == null catches cases where the variable is undefined.